examples/DrawingInTheAir/DrawingInTheAir_MotionActivated

```diff
@@ -0,0 +1,290 @@
+ *    cutting sample size in half.
+ *
+ * 2. Undersample. If we are sampling at 200Hz and the button is held for 1.2
+ *    seconds, then we'll have ~240 samples. Since we know now that each
+ *    sample, once compressed, will occupy 3 of our neuron's 128 bytes
+ *    (see #1), then we know we can only fit 42 of those 240 samples into a
+ *    single neuron (128 / 3 = 42.666). So if we take (for example) every 5th
+ *    sample until we have 42, then we should cover most of the sample window
+ *    and have some semblance of the original pattern. */
+void undersample(byte samples[], int numSamples, byte vector[])
+{
+    unsigned int vi = 0;
+    unsigned int si = 0;
+    unsigned int step = numSamples / samplesPerVector;
+    unsigned int remainder = numSamples - (step * samplesPerVector);
+
+    /* Centre sample window */
+    samples += (remainder / 2) * 3;
+    for (unsigned int i = 0; i < samplesPerVector; ++i) {
+        for (unsigned int j = 0; j < 3; ++j) {
+            vector[vi + j] = getAverageSample(samples, numSamples, si + j, step);
+        }
+
+        si += (step * 3);
+        vi += 3;
+    }
+}
+
+void readVectorFromIMU(byte vector[])
+{
+    byte accel[sensorBufSize];
+    int raw[3];
+
+    unsigned int samples = 0;
+    unsigned int i = 0;
+
+    /* Wait until Motion detection inturrupt flag  */
+    while (ledState == false)
+{
+    digitalWrite(11, LOW);  // Optional Beeper
+}
+    /* While Motion detection inturrupt flag being held... */
+    while (ledState == true) {
+
+        if (CurieIMU.accelDataReady()) {
+
+            CurieIMU.readAccelerometer(raw[0], raw[1], raw[2]);
+
+            /* Map raw values to 0-255 */
+            accel[i] = (byte) map(raw[0], IMULow, IMUHigh, 0, 255);
+            accel[i + 1] = (byte) map(raw[1], IMULow, IMUHigh, 0, 255);
+            accel[i + 2] = (byte) map(raw[2], IMULow, IMUHigh, 0, 255);
+
+            i += 3;
+            ++samples;
+
+            /* If there's not enough room left in the buffers
+            * for the next read, then we're done */
+            if (i + 3 > sensorBufSize) {
+                break;
+            }
+
+        }
+digitalWrite(11, HIGH);
+delay(50);
+digitalWrite(11, LOW);
+    }
+
+    undersample(accel, samples, vector);
+}
+
+void trainLetter(char letter, unsigned int repeat)
+{
+    unsigned int i = 0;
+
+    while (i < repeat) {
+        byte vector[vectorNumBytes];
+
+        if (i) Serial.println("And again...");
+
+        readVectorFromIMU(vector);
+        CuriePME.learn(vector, vectorNumBytes, letter - upperStart);
+
+        Serial.println("Got it!");
+        delay(1000);
+        ++i;
+    }
+}
+
+void trainLetters()
+{
+    for (char i = trainingStart; i <= trainingEnd; ++i) {
+        Serial.print("Initiate Motion and draw the letter '");
+        Serial.print(String(i) + "' in the air. Here is no button, CURIE ZERO MOTION DETECTION activation automatically after drawing the letter ");
+        Serial.println("as you are done.");
```

Comment:
I think maybe you intended to delete this line? The message doesn't make much sense as a whole now